front/stores/appStore.js

```diff
@@ -84,14 +88,50 @@ export const useAppStore = defineStore('app', {
   },
 
   actions: {
+    async syncEverything() {
+      if (!this.hasAuthToken) {
+        return
+      }
+      const async1 = this.fetchLatestAppVersion()
+      const async2 = this.fetchUISettings()
+      await Promise.all([async1, async2])
+      this.lastSync = new Date()
+    },
+
+    async fetchUISettings() {
+      const authTokenHash = await sha256(this.authToken)
+      const response = await new UISettingsRepository().getOne(authTokenHash)
+
+      if (response.data == null) {
+        return
+      }
+
+      const newValues = UISettingsTransformer.transformFromApi(response.data)
+
+      this.ui.darkTheme = newValues.darkTheme
+      this.ui.numberFormat = newValues.numberFormat
+      this.ui.lowerCaseAccountName = newValues.lowerCaseAccountName
+      this.ui.lowerCaseCategoryName = newValues.lowerCaseCategoryName
+      this.ui.lowerCaseTagName = newValues.lowerCaseTagName
+      this.ui.lowerCaseTransactionDescription = newValues.lowerCaseTransactionDescription
```

Comment:
I wish I could just do something like `this.ui = newValues` but I think it breaks local storage behaviour.